extensions/sequelize/src/__tests__/fixtures/models/scoped-task.model.ts

```diff
@@ -0,0 +1,36 @@
+import {Entity, model, property} from '@loopback/repository';
+
+/**
+ * Simplified Entity used for testing settings "scope" functionality
+ */
+@model({
+  settings: {
+    scope: {
+      limit: 1,
```

Comment:
Can you also add 1 more test case for where clause please ? That will help verify that use case too.

Reply:
Certainly!